src/commands/lambda/function.ts

```diff
@@ -100,6 +106,10 @@ const getLambdaConfig = async (
 const getLayerArn = (runtime: Runtime, settings: InstrumentationSettings, region: string) => {
   const layerName = RUNTIME_LAYER_LOOKUP[runtime]
   const account = settings.layerAWSAccount ?? DEFAULT_LAYER_AWS_ACCOUNT
+  const isGovCloud = region === 'us-gov-east-1' || region === 'us-gov-west-1'
```

Comment:
Since CLI can work automatically for new regions (i.e., layer arns not hard coded in a file), it might be better to only check against the prefix `us-gov`?